### DISCUSSION TITLE
Migrate to Free Version

### DISCUSSION THREAD

**Discussion by rangav** [thunderclient/thunder-client-support#1286] [Announcements]
Hi, 

We announced Subscriptions on `May 19th` -  https://github.com/rangav/thunder-client-support/releases/tag/v2.7.0

### Move To Free Version
If you want to move to the **free version** and stop using **git-sync feature** please follow the below steps

**Step1:** move the collections and environments folders to the default location below
https://github.com/rangav/thunder-client-support#storage

**Step2:** disable git sync settings in vs code ( see below image )
https://github.com/rangav/thunder-client-support#git-sync

**Step3:** restart vs code and you should see all the data.

### Free Version Limits
- Activity - No Limits
- Environments - Unlimited 
- Collections - Unlimited 
- Collection Runs - 30 per month
- No CLI

### Compare Plans
- https://www.thunderclient.com/pricing#compare

### Move To Another Tool
- [Convert to Postman](https://github.com/rangav/thunder-client-support#how-to-convert-to-postman-format) format and export json file

### Support
If you have any further questions contact using below link
https://www.thunderclient.com/contact

**Comment by chengkiang** [comment]
I read [this page](https://github.com/rangav/thunder-client-support#git-sync) a number of times and still cannot figure out how exactly to disable the Git Sync feature. It only explains Workspace vs Custom Location settings. And both Workspace and Custom Location are subscription features. 

Can you elaborate on how exactly to do this?

**Reply by rangav** [reply]
Disable Or Remove values for git related vscode settings 
<img width="1151" alt="Screenshot 2023-08-22 at 07 50 31" src="https://github.com/rangav/thunder-client-support/assets/8637550/a0a23a67-8478-4a9a-86f1-181a40023ecd">

**Comment by olcaybuyan** [comment]
@rangav you should add a button or setting that migrates users... I will simply uninstall this extension and move on. What a shame, this used to be a great product.

**Reply by rangav** [reply]
Default location might have existing data, so difficult to do automatically.

**Reply by riteshkohad** [reply]
Naaaa bro, you should have given a button to export existing collections to have people graceful exit from your tool. You misled people (at least me) by stating that Git Sync will be a paid feature, had I known before that my collections would be gone too and I will have a very strict environment I would have moved earlier.

**Reply by rangav** [reply]
Hey buddy, can't you follow the above simple 3 steps? if we do it automatically and some data goes missing, you will complain again.

**Comment by YoussefBoudiba** [comment]
> @rangav you should add a button or setting that migrates users... I will simply uninstall this extension and move on. What a shame, this used to be a great product.

I feel the exact same way..back to postman I guess..

**Comment by nesslopz** [comment]
I'm disappointed to learn about the recent change you've made here. The product was truly exceptional, and its move has left me quite sad. We do recognize the importance of generating revenue, but restricting an existing feature doesn't seem like a favorable approach. Perhaps considering limitations for new features only would be a more positive direction.

**Comment by rangav** [comment]
Hi All

- If we charge for our 1st version released in 2021 which does not have all the new features we developed in the last 2 years - Users will complain about how can you charge for such a basic software that does not have features.
- After developing all the NEW features in the last 2 years, if we want to charge now - You are complaining now how can we charge for existing features
- Some users complain about how can we charge for the X or Y feature.
- So no matter when we charge some users will complain.

It's easy to come up with an argument for not to charge for the software.

We have 2M+ downloads and got less than 500 dollars in GitHub sponsorship in 2 years.

Our team is no longer willing to work for free. So we have started charging users to use our software to cover our expenses.

---

If you need any further help in migration. Please contact us using the below link
https://www.thunderclient.com/contact
